docs/blog/welcome.md

```diff
@@ -1,3 +1,12 @@
+---
+author: 'Gao Ce'
+avatar: 'https://www.github.com/gaocegege.png'
+github: 'https://github.com/gaocegege'
+twitter: 'https://twitter.com/gaocegege'
+introduction: 'Do cool things that matter | Co-founder @tensorchord | Co-chair @kubeflow | ex-@Tencent, @bytedance, @caicloud'
+---
+<Author/>
```

Comment:
Then could you please move it to the last? Then I will merge it ASAP.

Thanks for your contribution! :tada: :+1: